msft-windows/msft-windows-rds-logon-report.ps1

```
@@ -0,0 +1,129 @@
+        Write-Host "No RDS logon events found in the last $DaysBack days."
+        $events = @()
+    } else {
+        Write-Host "Failed to query event log: $($_.Exception.Message)"
+        Stop-Transcript
+        exit 1
+    }
+}
+
+Write-Host "Found $($events.Count) events. Parsing..."
+
+$results = foreach ($event in $events) {
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'event' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)